packages/sdk/src/authentication/BasicAuthProvider.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AxiosRequestConfig } from "axios";
+import { IAuthProvider } from "./IAuthProvider";
+
+export class BasicAuthProvider implements IAuthProvider {
+  private config: BasicAuthConfig;
+
+  constructor(config: BasicAuthConfig) {
```

Comment:
what's the purpose of using a config object instead of defining 2 parameters (username/password)?

Reply:
Updated to use parameters instead of config object.